### ISSUE TITLE
supress second mousedown event after touchstart event

### ISSUE THREAD

**Issue by rfrei** [bbc/peaks.js#251]
I have discovered another minor problem and will mention it in another issue: konvas _touchstart event handler does not call preventDefault() and because of that we get a second mousedown event after the touchstart event. This leads to short repeats of audio while playing and touching on the waveform. I come back soon.

_Originally posted by @rfrei in https://github.com/bbc/peaks.js/pull/249#issuecomment-448233593_

What I found here is:
- konva calls for shape objects if the shape is listening for events evt.preventDefault() in _touchstart
- that makes sure no additional mouse events emulated by the browser will be dispatched after a touch event, peaks.js shows this behaviour if we touch a segments or pointer marker handle
- by touching the waveform it does not avoid handling of additional mouse events because the waveform is no shape and this leads to a second mousdown event and a short jump back while playing audio
- to avoid this we must call preventDefault() somewhere in MouseDragHandler.prototype.mouseDown
- the drawback is that if we call preventDefault() we can not scroll the page anymore by touching the waveform but only the waveform itself

In my opinion it is more important to avoid the audio jumps and call preventDefault()

What do you think?

for reference:
https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Supporting_both_TouchEvent_and_MouseEvent

**Comment by rfrei** [comment]
Note: the described behavior occurs only after aplying the konva patches for mouse events on mobile devices https://github.com/konvajs/konva/pull/517

**Comment by rfrei** [comment]
Ok, just found the right solution. I have to call preventDefault() on the touchend event, not the touchstart event because this is the end of the touch cycle. In this way everything works correctly. Scroll and zoom on mobile devices works and I have no audio glitches.

One thing I am sorry: with more time I should have included this in my first commit. As I said before this becomes important after using konva with patches for mouse events on mobile devices.
I have made a push request for this https://github.com/bbc/peaks.js/pull/252

Just another source of information:
https://www.html5rocks.com/en/mobile/touchandmouse/

**Comment by rfrei** [comment]
Thank you.

**Comment by chrisn** [comment]
@rfrei I have now published Peaks.js 0.9.14, since Konva has been updated with your changes. Please take a look!

**Comment by rfrei** [comment]
Thank you. I have tested it and #252 needs still be merged.

**Comment by rfrei** [comment]
Thanks again.
